### ISSUE TITLE
Swashbuckle applies a 5th hit, if Pangolier dashes through enemies

### ISSUE THREAD

**Issue by TheZett** [ValveSoftware/Dota2-Gameplay#826]
### Ability name

Swashbuckle

### Description

If Swashbuckle causes Pangolier to dash through enemies, it applies a 5th attack to them, instead of just applying the intended 4 attacks.

This behaviour has been in the game for quite a while, but it was never mentioned in any official documents; it is not mentioned in-game and no changelogs or patch notes mention it.

Is this behaviour intended or is it a bug?

Either the spell should get fixed to only apply 4 attacks, or the tooltip should get updated and state, that dashing through enemies applies a bonus attack on them.

### Example Match ID (and possibly Timestamp)

https://youtu.be/x78SZS7YUnI

**Comment by TheZett** [comment]
The attack is still applied when you are unable to attack (disarmed, ethereal, stunned) and even when having negative attack range (druid ult + troll stance change + swashbuckle on a Rubick), which prevents you from auto-attacking in general.

This makes me conclude that the 5th slash is not actually an auto-attack, but actually a 5th Swashbuckle slash instead.

**Comment by arthurnalves** [comment]
This should definitely  become a feature, if it isn't already.

**Comment by LokeLok** [comment]
#271 Same issue

**Comment by ptohidi** [comment]
I also agree. It does seem like an interesting feature, but if it's intended, it should be explicitly mentioned in the ability description.
